LICENSE.md

```diff
@@ -1,3 +1,13 @@
+## A few parts of this project are not in the public domain
```

Comment:
Adapted this format from USWDS:

https://github.com/uswds/uswds/blob/develop/LICENSE.md

Related: https://github.com/18F/open-source-policy/issues/36